packages/modules/src/modules/banners/choiceCardsButtonsBanner/ChoiceCardsButtonsBanner.stories.tsx

```diff
@@ -116,8 +116,11 @@ Blue.args = {
     tracking,
     choiceCardAmounts: {
         testName: 'Storybook_test',
-        variantName: 'Control',
-        amounts: {
+        variantName: 'CONTROL',
+        defaultContributionType: 'MONTHLY',
+        displayContributionType: ['ONE_OFF', 'MONTHLY', 'ANNUAL'],
```

Comment:
One thing I don't 100% understand: when would the contribution types in this array be different from the keys of the `amountsCardData` object?

Reply:
`defaultContributionType` and `displayContributionType` come from the contributionType JSON file which we're trying to deprecate and merge into the new Amounts JSON file. 

- `displayContributionType` - if the string is in the array, then the related tab in the amounts card displays (eg if a test has been setup to only show recurring contributions, excluding the single payment prices
- `defaultContributionType` - the tab amounts which should be displayed when the amounts card loads into the banner or epic

... does that make sense?